RELEASE.md

```diff
@@ -0,0 +1,53 @@
+# How to create a new `govmomi` Release
+
+## Verify `master` branch is up to date with the remote
+
+```bash
+git checkout master
+git fetch -avp
+git diff master origin/master
+
+# if your local and remote branches diverge run
+git pull origin/master
+```
+
+> ⚠️ **Note:** These steps assume `origin` to point to the remote
+> `https://github.com/vmware/govmomi` (respectively
+> `git@github.com:vmware/govmomi`).
+
+## Create the new Version Tag
+
+
+Create a new release tag adhering to the [semantic
+versioning](https://semver.org/) scheme:
+
+```bash
+# Example
+git tag -a v0.25.0 -m "Release v0.25.0"
+```
+
+## Push the new Tag
+
+```bash
+# Will trigger Github Actions Release Workflow
+git push origin --tags
```

Comment:
Maybe just the specific tag? `git push origin refs/tags/v0.25.0`

Reply:
Good catch!